app/boards/shields/kyria/kyria-layout.dtsi

```diff
@@ -0,0 +1,137 @@
+#include <physical_layouts.dtsi>
+
+/ {
+    splitkb_kyria_6col_layout: splitkb_kyria_6col_layout {
+        compatible = "zmk,physical-layout";
+        display-name = "6-column";
```

Comment:
I tend to favor the more human readable:

```suggestion
        display-name = "6 Column";
```

Reply:
Makes sense. Since Corne also had the same issue, I took the initiative to replace that in a new commit as well.